### DISCUSSION TITLE
Is this project dead?

### DISCUSSION THREAD

**Discussion by jonybekov** [visjs/vis-timeline#1645] [General]
I see it's been a year since this library released a new patch with bugfixes.

Is it still maintained?

**Comment by ProIntegritate** [comment]
I think they maintain it but sparsely, mostly necessary fixes i guess.

They were looking for people to maintain it 1-2 years ago, but not sure if they found anyone. This is the best and most feature rich timeline tool available **anywhere** and it would be a real shame to see if fall out of compatibility with browsers.

**Comment by mrpinkcat** [comment]
The number of unreviewed/uncommented PR make me sad 😢

**Comment by ebrensi** [comment]
Are there any comparable projects with a similar API?
